packages/components/src/components/listitem/listitem.utils.ts

```diff
@@ -0,0 +1,7 @@
+import { v4 as uuidv4 } from 'uuid';
+
+const generateTooltipId = () => `listitem-tooltip-popover-${uuidv4()}`;
+
+const generateNewListItemId = () => `listitem-tooltip-triggerid-${uuidv4()}`;
```

Comment:
issue: please fix this one as well

```suggestion
const generateListItemId = () => `listitem-tooltip-triggerid-${uuidv4()}`;
```